js/main.js

```diff
@@ -0,0 +1,91 @@
+var button = document.getElementById('buttonNews');
+button.onclick = function displayAndHide() {
+    var hide = document.getElementById('displayInfo');
+    if (hide.style.display === "") {
+        hide.style.display = 'none';
+    } else {
+        hide.style.display = '';
+    }
+}
+
+var submitBtn = document.querySelector('#submitBtn');
+submitBtn.addEventListener("click", checkName);
+
+function checkName(evt) {
+    evt.preventDefault();
+    var nameArea = document.querySelector("#nameArea");
```

Comment:
good that you define your selector variables inside the function not in the *global scope* - avoid global scope as much as possible.